WordPressKit/WordPressKit/WordPressComRestApi.swift

```diff
@@ -70,22 +80,30 @@ open class WordPressComRestApi: NSObject {
         sessionManager.requestSerializer = AFJSONRequestSerializer()
         return sessionManager
     }
-
-    public init(oAuthToken: String? = nil, userAgent: String? = nil) {
+    
+    convenience public init(oAuthToken: String? = nil, userAgent: String? = nil) {
+        self.init(oAuthToken: oAuthToken, userAgent: userAgent, backgroundUploads: false, backgroundSessionIdentifier: WordPressComRestApi.defaultBackgroundSessionIdentifier)
+    }
+    
+    public init(oAuthToken: String? = nil, userAgent: String? = nil, backgroundUploads: Bool = false, backgroundSessionIdentifier: String = WordPressComRestApi.defaultBackgroundSessionIdentifier) {
```

Comment:
@SergioEstevao Could you add some doc comments here with discussion about the background upload functionality?

Reply:
Will do!